azure-functions/number-generator-function/api.js

```diff
@@ -20,47 +24,53 @@ const getAPI = async (type) => {
   return response;
 };
 
-const postToAPI = async (apiEndpoint, apiData) => {
-  console.info('Azure functions - api.postToAPI');
-  if (!process.env.MULESOFT_API_UKEF_TF_EA_URL) {
+/**
+ * Post to MDM endpoints
+ * @param {String} endpoint MDM endpoints
+ * @param {Object} payload Payload
+ * @returns API endpoint invoke response
+ */
+const postToAPI = async (endpoint, payload) => {
+  console.info('Calling number generator');
```

Comment:
It's surprising that the function seems to allow any post to MDM, based on its name and docs, but the log message is specifically about the `/numbers` endpoint. Maybe we should move the log message to the `callNumberGenerator` function instead?

Reply:
Good spot, have re-phrased the console.info.